de.fu_berlin.inf.dpp.stf/sonar-project.properties

```diff
@@ -1,14 +1,18 @@
 sonar.projectName        = Saros/STF
 sonar.projectDescription = Saros Test Framework
-sonar.projectVersion     = 0.1.0.DEVEL
+sonar.projectVersion     = 14.11.28.DEVEL
```

Comment:
Please use an internal version as `0.1.0`. I see no reason to increase the version of the STF for each release.